src/styles/variables.sass

```diff
@@ -0,0 +1,49 @@
+// Favorite constants
+$gratio: 1.618
+$pi: 3.14159
+
+// Dimensions
+$radius: 2px
+$spacing: 24px
+$padding: 10px
+
+// Font properties
+$font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif
+$font-family-serif: Georgia, Cambria, 'Times New Roman', Times, serif
+$font-family-monospace: Monaco, Menlo, Consolas, 'Courier New', monospace
```

Comment:
Není to hardcoded do deklarací, ale je to vyčleněo sem, díky čemuž člověk může relevantní věci snadno měnit přímo z `variables` (kam jinam bys jako první šel, kdyby Ti grafik řekl, že se změnilo písmo?). Používá se to (nepřímo) mimo jiné v `global.sass`.